### ISSUE TITLE
Remove Webmaker logo from Teach site login pages

### ISSUE THREAD

**Issue by mmmavis** [mozilla/id.webmaker.org#407]
Re: https://github.com/mozilla/teach.mozilla.org/issues/1383#issuecomment-169686824

Can we conditionally hide Webmaker logo based on `client_id`? If so let's hide the logo if the redirect comes from teach-api (both production `wm_id_YX9X9DqK4d4fB9LuFbZy` and staging `wm_id_1H9nvpGGTQ4CkXGVn56p`).  

d(`･∀･)b

**Comment by Pomax** [comment]
Do we have a URL that shows off the wrong view right now?

@cade do we have anything in place that allows apps to register their own logo/branding? If so we might be able to use that instead.

**Comment by cade** [comment]
I’m pretty sure we can hotlink a logo within an iframe, right @cadecairos?

**Comment by cadecairos** [comment]
looool

**Comment by cadecairos** [comment]
@Pomax no, the Oauth provider doesn't have branding related fields for oauth clients

**Comment by mmmavis** [comment]
This happened to me a few times when I tried to tag Simon on Github.  i.e.,"at wex" vs @simonwex :wink: :smile:

**Comment by cade** [comment]
![o](https://cloud.githubusercontent.com/assets/3403/12308835/10f94e44-ba0d-11e5-8050-ae489feb59d4.jpg)

Let’s do this. :beers: :maple_leaf:

**Comment by Pomax** [comment]
damnit, I am on the wrong side of Canada =_=
